test/test_conditional_feature_importance.py

```diff
@@ -848,3 +850,43 @@ def test_cfi_reproducibility_with_rng(cfi_test_data):
     cfi_2.fit(X_train)
     vim_reproducibility = cfi_2.importance(X_test, y_test)
     assert np.array_equal(vim, vim_reproducibility)
+
+
+@pytest.mark.parametrize(
+    "n_samples, n_features, support_size, rho, seed, value, signal_noise_ratio, rho_serial",
+    [(500, 50, 5, 0.1, 0, 8.0, 4, 0.0)],
+    ids=["default data"],
+)
+def test_cfi_cv(data_generator):
+    """
+    Test that CFI with cross-validated estimator works as expected. In particular,
+     - Empirical FDP is below the target FDR level
+     - Power is above 0.8, which is an arbitrary threshold
+
+    Note: Although only the expected FDP should be controlled, in practice
+    the simulation setting is simple enough to satisfy this stronger condition.
+    """
+    X, y, important_features, not_important_features = data_generator
+
+    model = RidgeCV(alphas=np.logspace(-3, 3, 13))
+    cv = KFold(n_splits=5, shuffle=True, random_state=0)
+    estimators = [
+        clone(model).fit(X[train_index], y[train_index])
+        for train_index, _ in cv.split(X)
+    ]
+    cfi_cv = CFICV(
+        estimators=estimators,
+        cv=cv,
+        random_state=0,
+        n_jobs=5,
+    )
+    cfi_cv.fit_importance(X, y)
+    selected = cfi_cv.fdr_selection(fdr=0.05)
+
+    alpha = 0.05
+    tp = np.sum([int(i) in important_features for i in np.where(selected)[0]])
```

Comment:
this is a bit awkward, but could be addressed in future PR

Reply:
I changed it to use the fdp_power function instead.